android-app/app/src/main/java/de/dhbw/ka/se/fibo/utils/ApiUtils.java

```diff
@@ -0,0 +1,87 @@
+package de.dhbw.ka.se.fibo.utils;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.NetworkResponse;
+import com.android.volley.ParseError;
+import com.android.volley.Response;
+import com.android.volley.toolbox.HttpHeaderParser;
+import com.android.volley.toolbox.JsonRequest;
+import com.android.volley.toolbox.StringRequest;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Map;
+
+public class ApiUtils {
+
+    private static String getBaseURL() {
+        if (ActivityUtils.isEspressoTesting()) {
+            return "http://localhost:8000";
+        } else {
+            return "10.0.2.2:8000";
```

Comment:
missing protocol -> registration no longer works
```suggestion
            return "http://10.0.2.2:8000";
```